### ISSUE TITLE
'Extra' items not marked as extra when parting out into current document

### ISSUE THREAD

**Issue by SteveTheBrick** [rgriebl/brickstore#333]
When I part out a set into a new document all 'extra items' are marked as such. When I part it out into the current document all the extra items are just regular items - is this the intended behaviour?

Today I parted out all 24 mini sets from a used advent calendar into the current document. I was expecting to be able to sort the items by status and delete all the extra items but instead I'm left with a lot of work (consolidate everything and adjust quantities).

**Comment by paramecie** [comment]
> is this the intended behaviour?

Yes - the part out choices (CTRL+I, CTRL+I) and the part out from a lot in a document don't share the same options.

I already proposed - somewhere - that those share the same options in the main options/settings...
Could be nice, I agree.
